packages/sanity/src/desk/panes/document/DocumentOperationResults.tsx

```diff
@@ -1,35 +1,34 @@
 import {useToast} from '@sanity/ui'
 import React, {memo, useEffect, useRef} from 'react'
 import {useDocumentPane} from './useDocumentPane'
-import {useDocumentOperationEvent} from 'sanity'
+import {useDocumentOperationEvent, useTranslation} from 'sanity'
 import {usePaneRouter} from '../../components'
+import {deskLocaleNamespace} from '../../i18n'
 
 function getOpErrorTitle(op: string): string {
```

Comment:
Would be nice to rename this to indicate that it returns a key and not an actual title. Additionally, we could import and use the `DeskLocaleResourceKeys` type as the return value for better type safety

Reply:
I switched to using contexts and got rid of these methods altogether